docs/concepts/operator.md

```diff
@@ -262,6 +265,16 @@ This section highlights the most prominent fields:
   The `allowedHosts` field is explained [here](https://github.com/gardener/dashboard/blob/master/docs/operations/webterminals.md#configuration).
   The `container` section allows you to specify a container image and a description that should be used for the web terminals.
 
+##### [Gardener Discovery Server](https://github.com/gardener/gardener-discovery-server)
+`.spec.virtualCluster.gardener.gardenerDiscoveryServer` serves a few configuration options for the discovery server.
+- `hostname`: This is the hostname that will serve the OIDC discovery documents of the shoot clusters.
+  In addition to that the value will be used for calculating the service account issuer of a shoot cluster in the format `https://<hostname>/projects/<project-name>/shoots/<shoot-uid>/issuer`.
+  This configuration applies for all seeds registered with the Garden cluster. Once set it should not be modified.
+- `tlsSecretName`: This is the name of the secret that will be used by the discovery server for serving TLS.
+  Usually this secret contains a ceritificate issued for the `hostname` configured in the previously mentioned field.
+  The secret should contain two data keys `tls.key` and `tls.crt` containing the private key and the TLS certificate respectively.
+  If not provided, a self-signed certificate will be generated.
```

Comment:
Why would we allow specifying "an external" certificate here? `gardener-operator` can manage this and its rotation with the already existing infrastructure. I propose to remove this from the API.

Reply:
See https://github.com/gardener/gardener/pull/9746#discussion_r1609308676